sync/sync0sync.cc

```diff
@@ -162,16 +162,16 @@ Q.E.D. */
 
 /** The number of iterations in the mutex_spin_wait() spin loop.
 Intended for performance monitoring. */
-static int64_t mutex_spin_round_count = 0;
+static counter_t mutex_spin_round_count{};
 
 /** The number of mutex_spin_wait() calls.  Intended for performance monitoring. */
-static int64_t mutex_spin_wait_count = 0;
+static counter_t mutex_spin_wait_count{};
 
 /** The number of OS waits in mutex_spin_wait().  Intended for performance monitoring. */
-static int64_t mutex_os_wait_count = 0;
+static counter_t mutex_os_wait_count{};
 
 /** The number of mutex_exit() calls. Intended for performance monitoring. */
-int64_t mutex_exit_count = 0;
+counter_t mutex_exit_count{};
```

Comment:
@sunbains : I could only find these for performance monitoring(text search of performance monitoring in the code). Let me know if there are other places as well.

Reply:
They are useful where the frequency is very high. It's not critical that all counters should be replaced. A lazy effort is good enough.

Reply:
Sounds good!